deploy/helms/legion-core/templates/jenkins/jenkins-pvc.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.jenkins.enabled }}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: "{{ .Release.Name }}-jenkins-master-home"
+  labels:
+    app: "{{ .Release.Name }}-jenkins-master-home"
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+spec:
+  accessModes:
+    - {{ .Values.jenkins.persistence.accessMode | quote }}
```

Comment:
Do we really need a parameter for this?

Reply:
I think potentially other pod in system may require access to than PVC.